### PR TITLE
wot-author stories

### PR DESCRIPTION
**Pull request by mlsteele**
Get some stories going. Not hooked up. ~Looks like garbage on mobile~, but I would find it helpful for someone to look for structural issues in parallel with debugging mobile.

image
image

### CODE REVIEW

File: shared/profile/wot-author/index.tsx [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+        <Kb.LabeledInput
+          placeholder="Your claim"
+          hoverPlaceholder={'Write how you met them and what experiences you shared with them.'}
+          multiline={true}
+          rowsMin={9}
+          autoFocus={true}
+          maxLength={statementLimit}
+        />
+        <Kb.Text type="BodySmall" style={styles.approveNote}>
+          cecileb will be able to approve or suggest edits to what you’ve written.
```
Comment (jakob223) [inline shared/profile/wot-author/index.tsx]: Why is Cècile in charge of all this?

**Comment by mlsteele** [comment]
> ^ is this behavior and the "other" behavior part of a different ticket or this one? 
TBD but they are not part of this PR.

**Comment by mlsteele** [comment]
Now with mobile support and a small radio-button change.

File: shared/common-adapters/radio-button.native.tsx [hunk 2]
```diff
@@ -1,10 +1,12 @@
 import React from 'react'
 import ClickableBox from './clickable-box'
+import {Box2} from './box'
 import Text from './text'
 import * as Styles from '../styles'
 import {Props} from './radio-button'
 
 const Kb = {
+  Box2,
```
Comment (jakob223) [inline shared/common-adapters/radio-button.native.tsx:9]: unused?

File: shared/profile/wot-author/index.tsx [hunk 3]
```diff
@@ -130,45 +133,78 @@ const VerificationChoice = (props: {
...
+    <Kb.Text type="Body" style={textStyle}>
+      Do not choose this option
+    </Kb.Text>
+  )
   let color: string = Styles.globalColors.white
   switch (props.verificationType) {
     case 'in_person':
       text = (
-        <Kb.Text type="Body">
+        <Kb.Text type="Body" style={textStyle}>
```
Comment (jakob223) [inline shared/profile/wot-author/index.tsx:146]: I'm unconvinced by all this `text = <Kb.Text....` business.

How about 
```
let text: React.ReactNode
...
text = <>{props.voucheeUsername} told me their username <Kb.Text type="BodyBold">in person</Kb.Text></>
...
text = `${props.voucheeUsername} told me their username over video`
...
etc
...
label={<Kb.Text type="Body" style={Styles.globalStyles.flexOne}>{text}</Kb.Text>}
```

Comment (mlsteele) [reply shared/profile/wot-author/index.tsx:146]: fragment to the rescue! That looks better.

**Comment by mlsteele** [comment]
merging over golang test failure. Might regret this.

**Pull request merged**